ResNet_constructor.ipynb

```diff
@@ -0,0 +1,1393 @@
+{
```

Comment:
<p>I see no reason to keep this block</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bestetc/May/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Reply:
<p>agreed</p>